Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -177,6 +180,22 @@ class BocList
         selectedRows.OnSelectionChanged (bocList, false);
       }
     });
+
+    if (row.classList.contains("hasValidationRow")){
+      let validationRow = row.nextSibling as HTMLElement;
+
+      validationRow.addEventListener('click', (evt) =>
+      {
+        let hasSelectionChanged = BocList.OnRowClick(evt, bocList, validationRow, selectorControl);
+
+        if (hasSelectionChanged)
+        {
+          row.dispatchEvent(new Event("click"));
+          let selectedRows = BocList._selectedRows[bocList.id]!;
+          selectedRows.OnSelectionChanged(bocList, false);
+        }
```

Comment:
Not sure if that's not duplicate logic. I see that you dispatch the click event on the "parent" row and then also do all the things yourself, too. It seems to me that all we need to do is perform an unconditional disapatch onto the parent row and we're done.